packages/amplify_auth_plugin_interface/lib/amplify_auth_plugin_interface.dart

```diff
@@ -130,4 +130,9 @@ abstract class AuthPluginInterface extends AmplifyPluginInterface {
   Future<List<AuthDevice>> fetchDevices() {
     throw UnimplementedError('fetchDevices() has not been implemented.');
   }
+
+  /// Deletes the authenticated [AuthUser].
```

Comment:
This is where we need good docs. In big, bold letters: iOS only, cannot be called from Android.